### PR TITLE
chore: add an easy way to bump the version

### PR DESCRIPTION
**Pull request by henryiii**
This seems to be the replacement for the unmaintained bump2version, which is a fork of the unmaintained bumpversion...

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -104,3 +104,11 @@ commands =
     coverage html -d {toxworkdir}/htmlcov -i
     python -m diff_cover.diff_cover_tool --compare-branch {env:DIFF_AGAINST:origin/main} {toxworkdir}/coverage.xml
 depends = {py312, py311, py310, py39, py38, py37, pypy37, pypy38, pypy39}{,-min}, path
+
+[testenv:bump]
+description = bump versions, pass major/minor/patch
+skip_install = true
+setenv =
```
Comment (gaborbernat) [inline tox.ini:111]: set_env

File: tox.ini [hunk 2]
```diff
@@ -104,3 +104,11 @@ commands =
...
+deps = bump-my-version
```
Comment (gaborbernat) [inline tox.ini:112]: Add lower bound version spec

**Review by gaborbernat — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (henryiii) [reply tox.ini:111]: :facepalm: Ahh, I should have noticed that.

File: tox.ini (hunk 2 above)
Comment (henryiii) [reply tox.ini:112]: Okay with something recent? Given it's internal, I'd assume yes. (0.10 came out yesterday I think)

File: tox.ini (hunk 1 above)
Comment (henryiii) [reply tox.ini:111]: Ahh, I was copying the examples above, which also use the older format. Maybe we should move to the newer more readable forms all at once?

Comment (gaborbernat) [reply tox.ini:111]: Use https://github.com/tox-dev/tox-ini-fmt that auto upgrades

Comment (henryiii) [reply tox.ini:111]: That didn't add the underscores, all fields are the same after running it. It just moved things around and added some indentation. And broke this:

```diff
-depends = {py312, py311, py310, py39, py38, py37, pypy37, pypy38, pypy39}{,-min}, path
+depends =
+    -min}
+    path
+    py310
+    py311
+    py37
+    py38
+    py39
+    pypy37
+    pypy38
+    pypy39}{
+    {py312
```

Comment (gaborbernat) [reply tox.ini:111]: image

Are you using the latest?

Comment (gaborbernat) [reply tox.ini:111]: Fixed https://github.com/tox-dev/tox-ini-fmt/commit/69ba5f57bfbd234f3de4d15275e8122526b4d80f 👍

File: tox.ini [hunk 3]
```diff
@@ -106,3 +106,12 @@ commands =
 depends =
     path
     {py312, py311, py310, py39, py38, py37, pypy39, pypy38, pypy37}{, -min}
+
+[testenv:bump]
+description = bump versions, pass major/minor/patch
+skip_install = true
+deps =
+    bump-my-version
```
Comment (henryiii) [inline tox.ini:114]:
```suggestion
    bump-my-version >=0.10
```

**Comment by henryiii** [comment]
pre-commit.ci autofix

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**